circle.yml

```diff
@@ -12,11 +12,16 @@ test:
   override:
     - docker run -it -v `pwd`/test.py:/tmp/test.py quay.io/syncano/python-codebox python /tmp/test.py
     - docker run -it -v `pwd`/test.py:/tmp/test.py quay.io/syncano/python-codebox python27-lib4.2 /tmp/test.py
+
     - docker run -it -v `pwd`/test.py:/tmp/test.py quay.io/syncano/python-codebox python27-lib5.0 /tmp/test.py
     - docker run -it -v `pwd`/test_v50.py:/tmp/test.py quay.io/syncano/python-codebox python /tmp/test.py
+
     - docker run -it -v `pwd`/test_v42.py:/tmp/test.py quay.io/syncano/python-codebox python27-lib4.2 /tmp/test.py
     - docker run -it -v `pwd`/test_v50.py:/tmp/test.py quay.io/syncano/python-codebox python27-lib5.0 /tmp/test.py
 
+    - docker run -it -v `pwd`/test.py:/tmp/test.py quay.io/syncano/python-codebox python3 /tmp/test.py
+    - docker run -it -v `pwd`/test_v50.py:/tmp/test.py quay.io/syncano/python-codebox python3-lib5.0 /tmp/test.py
```

Comment:
Me just thinking that we can have some specific python3 code -> to make sure that python3 is actually python3 :) 

docker run -it -v `pwd`/test_py3.py:/tmp/test_py3.py quay.io/syncano/python-codebox python3 /tmp/test_py3.py

And the test_py3.py file should contain a code which for sure will be run in python 3, a not in python 2 :)

But I'm just thinking ;) loudly.